proto/config.go

```diff
@@ -31,6 +32,12 @@ import (
 // NodeID is a custom type for a cockroach node ID. (not a raft node ID)
 type NodeID int32
 
+// String implements the fmt.Stringer interface.
+// It is used to format the ID for use in Gossip keys.
+func (n NodeID) String() string {
```

Comment:
This might have unintended interactions with `fmt` (e.g. if "%08d" is ever used to format a `NodeID`). Probably only affects log messages, but just wanted to point it out.